tensorflow/lite/tools/evaluation/tasks/README.md

```diff
@@ -2,20 +2,19 @@
 
 This page describes how you can check the accuracy of quantized models to verify that any degradation in accuracy is within acceptable limits.
 
+## Accuracy & correctness
+TensorFlow Lite has two types of tooling to measure how accurately a delegate behaves for a given model: Task-Based and Task-Agnostic. 
+
+For more information visit the [Tools for Evaluation](https://www.tensorflow.org/lite/performance/delegates#tools_for_evaluation) page.
```

Comment:
Point to this instead, for the exact section: https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness